tests/valid/struct_from_json.test.w

```diff
@@ -263,9 +263,13 @@ bring "./subdir/structs_2.w" as otherExternalStructs;
 
 enum Color { red, green, blue }
 
+/// MyStruct docs
```

Comment:
Is it possible to support @doctags?

Reply:
Can you be more specific? I'm not sure what you have in mind

Reply:
I took a guess and added some code to the example to test the case when docstrings are multi-line or have doctags.

Reply:
I meant that it will be useful to be able to reflect on doctags directly:

```
@type(Foo).doctags["example"]
```

Reply:
I meant that it will be useful to be able to reflect on doctags directly:

```
@type(Foo).doctags["example"]
```

Reply:
Ah, got it. Let me try adding that in a separate PR